src/components/Sections/SectionsProgress.tsx

```diff
@@ -22,19 +34,79 @@ const SectionsProgress: React.FC<SectionsProgressProps> = ({
     )
   }
 
-  const getProgress = ({ doneRequired, doneNonRequired, totalSum, valid }: ProgressType) => {
+  const getIsStrictSection = (sectionCode: string) =>
+    sectionCode === firstStrictInvalidPage?.sectionCode
+
+  const setIsAfterStrict = (sectionCode: string) => {
+    if (!isAfterStrict)
+      isAfterStrict = firstStrictInvalidPage === null ? true : getIsStrictSection(sectionCode)
```

Comment:
could just be `firstStrictInvalidPage === null || getIsStrictSection(sectionCode)`, but that's stylistic pref, not needed change

Reply:
I don't know if that's the same... I needed to know if the section is **after** the strict not if it is the strict

Reply:
Ah! Oops, I understood now. Sorry. Will change to that, it looks better indeed